packages/webapp/src/pages/Connection/Syncs.tsx

```diff
@@ -208,8 +208,11 @@ export default function Syncs(props: SyncsProps) {
                                         )}
                                     </div>
                                     <div className="relative interact-with-sync">
-                                        <EllipsisHorizontalIcon className="flex h-5 w-5 cursor-pointer" onClick={() => toggleDropdown(sync.id)} />
-                                        {openDropdownId === sync.id && (
+                                        <EllipsisHorizontalIcon
+                                            className="flex h-5 w-5 cursor-pointer"
+                                            onClick={() => toggleDropdown(`${sync.id}${JSON.stringify(sync.models)}`)}
+                                        />
+                                        {openDropdownHash === `${sync.id}${JSON.stringify(sync.models)}` && (
```

Comment:
can we have a hash function or even maybe a constant to document what's this hash is?

Reply:
Thanks, updated with https://github.com/NangoHQ/nango/pull/1785/commits/f0415abe26dff5ce2401cc5676f1c1e4b1cd244a